packages/frontend/core/src/components/affine/auth/user-plan-button.tsx

```diff
@@ -35,17 +36,13 @@ export const UserPlanButton = () => {
   }, [subscriptionService]);
 
   const setSettingModalAtom = useSetAtom(openSettingModalAtom);
-  const handleClick = useCallback(
-    (e: React.MouseEvent<HTMLDivElement>) => {
-      e.stopPropagation();
-      setSettingModalAtom({
-        open: true,
-        activeTab: 'plans',
-        scrollAnchor: 'cloudPricingPlan',
-      });
-    },
-    [setSettingModalAtom]
-  );
+  const handleClick = useCatchEventCallback(() => {
```

Comment:
[![abandonship](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/WjlTemxb6oA4PgZFmj08/9ad7d584-1c9e-43ae-8e60-56fe2bf7d5ca/abandonship.gif)](https://app.graphite.dev//settings/meme-library?org=toeverything)

I was supposed to add a todo for it, but, well...

Reply:
looks like there are quite some that not fall into the scope of `useCatchEventCallback` when I am doing the refactor lol